docs/content/latest/reference/configuration/yb-master.md

```diff
@@ -455,7 +455,7 @@ On a per-table basis, the [`CREATE TABLE ... WITH TABLETS = <num>`](../../../api
 
 The number of shards per YB-TServer for each YSQL table when a user table is created.
 
-Default: `8`
+Default: `-1` (server internally sets default value). For servers with two or less CPU cores, then the default value is `2`. For servers with three or four CPU cores, the default value is 4. Beyond 4 cores, the default value is `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of 1.
```

Comment:
Also, is the difference between this and YCQL intentional? YCQL jumps straight to `8` for 3+ cores.

```suggestion
Default: `-1` (server internally sets default value). For servers with up to two CPU cores, the default value is `2`. For servers with three or four CPU cores, the default value is `4`. Beyond four cores, the default value is `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of `1`.
```

Reply:
Yes, it is intentional was based on a slack discussion involving Kannan and Karthik.

Reply:
Great. Just double-checking. Thanks!